app/src/main/java/org/fossasia/openevent/api/APIClient.java

```diff
@@ -37,6 +48,15 @@ public APIClient() {
                 .setConverter(new GsonConverter(gson))
                 .setEndpoint(Urls.BASE_GET_URL_ALT)
                 .setLogLevel(RestAdapter.LogLevel.FULL)
+                .setErrorHandler(new ErrorHandler() {
+                    @Override
+                    public Throwable handleError(RetrofitError cause) {
+                        ErrorDialogEvent errEvent = new ErrorDialogEvent(null, null);
+                        errEvent.ErrorHandlerFunction(cause, errEvent);
```

Comment:
I haven't looked at all the other issues you pointed, will do in a bit. But for this one, I've done this, look at these three images: [This](http://i.imgur.com/Bw41D12.png), [this](http://i.imgur.com/8AgNo3h.png) and [this](http://i.imgur.com/lebbW8J.png).

The first one is of the handleError function in APIClient, the next two are of HandleErrorDialogEvent.java (I changed the name).

Is this okay?